### PR TITLE
tier1, Add bridge-marker tier1 automation test

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
this PR adds a script that runs bridge-marker e2e test on CNAO cluster

**Special notes for your reviewer**:
This PR depends on bridge-marker [PR](https://github.com/kubevirt/bridge-marker/pull/20) merger.
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RamLavi** [comment]
/hold

wait for bridge-marker [PR](https://github.com/kubevirt/bridge-marker/pull/20) to be merged

File: automation/check-patch.e2e-bridge-marker-functests.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+main() {
+    # Setup CNAO and artifacts temp directory
+    source automation/check-patch.setup.sh
+    cd ${TMP_PROJECT_PATH}
+
+    # Spin-up ephemeral cluster with latest CNAO
+    # this script also exports KUBECONFIG, and fetch $COMPONENT repository
+    COMPONENT="bridge-marker" source automation/components-functests.setup.sh
```
Comment (qinqon) [inline automation/check-patch.e2e-bridge-marker-functests.sh:27]: We should set `KUBEVIRT_PROVIDER=external`

**Review by qinqon — CHANGES_REQUESTED** [review]

File: automation/check-patch.e2e-bridge-marker-functests.sh (hunk 1 above)
Comment (RamLavi) [reply automation/check-patch.e2e-bridge-marker-functests.sh:27]: see [comment](https://github.com/kubevirt/bridge-marker/pull/20#discussion_r514249741) in bridge-marker repo PR

**Comment by RamLavi** [comment]
/retest
unrelated flaky

**Comment by RamLavi** [comment]
/hold

we should make sure this script is not broken by  #635 changes

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

**Pull request merged**